Essentials/src/main/java/net/ess3/api/events/LocalChatSpyEvent.java

```diff
@@ -1,115 +1,30 @@
 package net.ess3.api.events;
 
+import net.essentialsx.api.v2.ChatType;
+import net.essentialsx.api.v2.events.ChatEvent;
 import org.bukkit.entity.Player;
-import org.bukkit.event.Cancellable;
-import org.bukkit.event.Event;
 import org.bukkit.event.HandlerList;
 
-import java.util.IllegalFormatException;
 import java.util.Set;
 
 import static com.earth2me.essentials.I18n.tl;
 
 /**
- * Fired when a player uses local chat.
+ * Fired when a player uses local chat and message is being sent to spies
  */
-public class LocalChatSpyEvent extends Event implements Cancellable {
+public class LocalChatSpyEvent extends ChatEvent {
     private static final HandlerList handlers = new HandlerList();
-    private final Player player;
-    private final Set<Player> recipients;
-    private boolean cancelled = false;
-    private String message;
-    private String format;
 
-    public LocalChatSpyEvent(final boolean async, final Player who, final String format, final String message, final Set<Player> players) {
-        super(async);
-        this.format = tl("chatTypeLocal").concat(tl("chatTypeSpy")).concat(format);
-        this.message = message;
-        recipients = players;
-        player = who;
+    public LocalChatSpyEvent(final boolean async, final Player player, final String format, final String message, final Set<Player> recipients) {
+        super(async, ChatType.LOCAL, player, format, message, recipients);
```

Comment:
```suggestion
        super(async, ChatType.SPY, player, format, message, recipients);
```

_(note for myself later)_